.vscode/settings.json

```diff
@@ -0,0 +1,22 @@
+{
+  "javascript.validate.enable": true,
+  // Enable editor validation for graphql files
+  "eslint.validate": [
+    "javascript",
+    "javascriptreact",
+    "typescript",
+    "typescriptreact",
+    "graphql"
+  ],
+  "stylelint.configOverrides": {
+    "rules": {
+      "csstools/value-no-unknown-custom-properties": [true, {
+        "importFrom": [
+          "node_modules/@commercetools-frontend/ui-kit/materials/custom-properties.css"
+        ]
+      }]
+    }
+  },
+  "typescript.tsdk": "node_modules/typescript/lib",
+  "vscode-vale.path": "${workspaceFolder}/node_modules/.bin/commercetools-vale"
```

Comment:
Note: this requires the [vscode vale extension](https://github.com/testthedocs/vscode-vale) in order to work.